backend/apps/cover_letters/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from apps.candidates.models import Candidate
+from apps.vacancies.models import Vacancy
+from django.utils.translation import gettext_lazy as _
+from apps.cover_letters.storages import OverwriteStorage
+
+
+def content_file_name(instance, filename):
+    file_path = '/'.join(['cv', instance.owner.full_name, filename])
+    return file_path
+
+
+class CoverLetter(models.Model):
```

Comment:
CV stands for Curriculum Vitae, not a Cover Letter.

Reply:
fixed